doc/changelog.md

```diff
@@ -2,6 +2,9 @@
 
 <h3>New features</h3>
 
+* Automatic differentiation of `catalyst.accerate`-ed functions.
+  [(#920)](https://github.com/PennyLaneAI/catalyst/pull/920)
```

Comment:
Hey Erick, just a friendly reminder that there is a changelog entry pre-written in the epic (would appreciate also adding this example as a test to ensure it works, if it is not added already!)

Reply:
Added example from the epic and used this changelog entry. :)